<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
background-color: rgb(31, 149, 239);
}

QPushButton {
	color:white;
	height: 30px;
	border:none;
	border-radius:10px;
}

QPushButton:checked{
background-color: #F5FAFE;
color: #1F95EF;
font-weight:bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/newPrefix/icon/profile_pic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashboard_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/dashboard_white.png</normaloff>
             <normalon>:/newPrefix/icon/dashboard.png</normalon>:/newPrefix/icon/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profile_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/profile_white.png</normaloff>
             <normalon>:/newPrefix/icon/profile.png</normalon>:/newPrefix/icon/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="messages_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/messages_white.png</normaloff>
             <normalon>:/newPrefix/icon/messages.png</normalon>:/newPrefix/icon/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notifications_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/notifications_white.png</normaloff>
             <normalon>:/newPrefix/icon/notifications.png</normalon>:/newPrefix/icon/notifications_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/settings_white.png</normaloff>
             <normalon>:/newPrefix/icon/settings.png</normalon>:/newPrefix/icon/settings_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>291</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/newPrefix/icon/log_out_white.png</normaloff>:/newPrefix/icon/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="icon_name_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);
	color: #white;
}
QPushButton {
	color:white;
	text-align:left;
	height: 30px;
	border:none;
	padding-left:10px;
	border-top-left-radius:10px;
	border-bottom-left-radius:10px;
}
QPushButton:checked{
background-color: #F5FAFE;
color: #1F95EF;
font-weight:bold;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/newPrefix/icon/profile_pic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Sidebar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashboard_2">
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/dashboard_white.png</normaloff>
             <normalon>:/newPrefix/icon/dashboard.png</normalon>:/newPrefix/icon/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profile_2">
           <property name="text">
            <string>Profile</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/profile_white.png</normaloff>
             <normalon>:/newPrefix/icon/profile.png</normalon>:/newPrefix/icon/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="messages_2">
           <property name="text">
            <string>Messages</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/messages_white.png</normaloff>
             <normalon>:/newPrefix/icon/messages.png</normalon>:/newPrefix/icon/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notifications_2">
           <property name="text">
            <string>Notifications</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/notifications_white.png</normaloff>
             <normalon>:/newPrefix/icon/notifications.png</normalon>:/newPrefix/icon/notifications_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_2">
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/newPrefix/icon/settings_white.png</normaloff>
             <normalon>:/newPrefix/icon/settings.png</normalon>:/newPrefix/icon/settings_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>291</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>Sign Out</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/newPrefix/icon/log_out_white.png</normaloff>:/newPrefix/icon/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="header_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="menu">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/newPrefix/icon/menu.png</normaloff>:/newPrefix/icon/menu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>266</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/newPrefix/icon/search.png</normaloff>:/newPrefix/icon/search.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>266</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_15">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/newPrefix/icon/image (4).png</normaloff>:/newPrefix/icon/image (4).png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <widget class="QWidget" name="dashboard_page">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>220</y>
             <width>231</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dashboard page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="profile_page">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>220</y>
             <width>231</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Profile  page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="messages_page">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>210</y>
             <width>231</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Messagers  page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="notifications_page">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>230</y>
             <width>231</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notifications  page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="settings_page">
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>280</y>
             <width>231</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Settings page</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifications_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifications_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messages_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>messages_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profile_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>profile_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profile_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>profile_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messages_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>messages_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifications_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifications_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>673</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
